### DISCUSSION TITLE
How to open Whisker Menu from xfce4 in i3wm?

### DISCUSSION THREAD

**Discussion by therealmaxmoon** [i3/i3#6117] [Q&A] [answered]
Is there a way to get this or similar menu to start?

The reason is that I somehow miss a sorted overview by categories on the applications, which are installed, like games, which I never know if there are even games install on a Linux distribution, because I use i3wm asap after a fresh os installation.

Just executing xfce4-popup-whiskermenu brings the error: Failed to send D-Bus message.

![screenshot-2024_06_23-10_47_17](https://github.com/i3/i3/assets/18206603/1c072872-5818-4b59-a567-23c813a0ad7e)

**Comment by okraits** [comment]
https://gitlab.xfce.org/panel-plugins/xfce4-whiskermenu-plugin/ says you need to execute `xfce4-popup-whiskermenu`. So you can execute that command with a keybinding if you like.

**Reply by therealmaxmoon** [reply]
Like I wrote in my first post: If I execute xfce4-popup-whiskermenu, I get an error. I am adding a screenshot, because it looks like the information can be missed.

**Reply by okraits** [reply, marked as answer]
I'm sorry, I missed it indeed. I could imagine that it needs xfce4-panel to be running. You can start xfce4-panel and see if that fixes the issue.

**Reply by therealmaxmoon** [reply]
It really needs the panel. But okay, I will not use it very often, so I can open the panel for a minute, even if it looks funny to have it open in i3 :D

Thanks

**Reply by Failrule** [reply]
> It really needs the panel. But okay, I will not use it very often, so I can open the panel for a minute, even if it looks funny to have it open in i3 :D
> 
> Thanks

Did you decide to use i3 instead of xfce because you wanted use the features that i3 offers? (like less clicks or more efficient) you could use some thing like rufus (it categorize apps too) moreover polybar has an nice look and it's a useful panel with a lot of modules.
